### DISCUSSION TITLE
License: device limit?

### DISCUSSION THREAD

**Discussion by actuallymentor** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#558] [Q&A]
Hello!

I'm considering buying the pro version (was going to make a CLI script but this looks prettier) but have a bunch of devices.

Is there a device limit to the license? Can I consider this a "family" license that I can share with my family/friends up to a point?

**Comment by MatthiasKerbl** [comment]
> Hello!
> 
> I'm considering buying the pro version (was going to make a CLI script but this looks prettier) but have a bunch of devices.
> 
> Is there a device limit to the license? Can I consider this a "family" license that I can share with my family/friends up to a point?

Hi @actuallymentor,

If you buy an AlDente Pro license, you can use it on up to three users.

Cheers,
Matthias

**Reply by actuallymentor** [reply]
Users or devices? Either way can I convince you to up it to something like 10?

I'm generally a power user. The biggest reason to not just script this is convenience, but at €24 it's not an impulse buy.

From my perspective there would be no downside to making a license have 10 devices or so.

It's the kind of software where you wouldn't lose buyers. It's such a cool niche product that niche users want it, "normal" users like my family will never buy it for themselves.

So in my view a more permissive license would make buying more of a nobrainer for people like me (so I can force it on my family and feel good about myself) and you wouldn't lose customers, since the people I'd force-onboard would never spend that kind of money on this kind of product.

**Reply by StevenCurran** [reply]
You just said you wanted 10 keys to share to your friends? Devs have to make money too, don't be so damn cheap.

**Reply by actuallymentor** [reply]
Be nice Steven :)

I'm a dev too, that is why I'm saying this.

Public conversation rarely gets better by shaming someone for their input.

**Reply by MatthiasKerbl** [reply]
> Users or devices? Either way can I convince you to up it to something like 10?
> 
> I'm generally a power user. The biggest reason to not just script this is convenience, but at €24 it's not an impulse buy.
> 
> From my perspective there would be no downside to making a license have 10 devices or so.
> 
> It's the kind of software where you wouldn't lose buyers. It's such a cool niche product that niche users want it, "normal" users like my family will never buy it for themselves.
> 
> So in my view a more permissive license would make buying more of a nobrainer for people like me (so I can force it on my family and feel good about myself) and you wouldn't lose customers, since the people I'd force-onboard would never spend that kind of money on this kind of product.

Hi @actuallymentor,

Our license is user-based. Since most people only use one user, they can use it on up to three devices. In my opinion, three users per license is already above average since most apps start with single or double user licenses. However, we will probably update our payment options for power users and companies which need more licenses.

Thank you for your input and hope you understand. :)

Cheers,
Matthias

**Reply by actuallymentor** [reply]
Thanks for your response!

Do I understand that with user you mean "apple account"? In that case my use case would be mostly covered. I've got 3 devices associated to my own Apple account purely for testing purposes.

Personally I always appreciate Family Licences. Software like little snitch and the like are examples for which I tend to just opt for more permissive licenses. Price is not the main issue, it's more a convenience factor.

**Reply by MatthiasKerbl** [reply]
Exactly, with users I mean macOS user accounts.

**Reply by actuallymentor** [reply]
Thanks for the clarification, I thought you meant apple accounts and not device users.

**Reply by Ming-D-W** [reply]
Will switching device users or resetting the mac consume a quota？

**Reply by MatthiasKerbl** [reply]
Every time you activate AlDente Pro with a license will use one quota.

Cheers,
Matthias

**Comment by pawanjenu** [comment]
@MatthiasKerbl , need for info on license. 
let's say, if I have 3 devices, and I activate pro on all 3 devices, and later on If I guy a new laptop, will I be able to transfer the license to the new device? 
or I will have to buy a new license?

**Reply by MatthiasKerbl** [reply]
Hi @pawanjenu,

Yes, you will be able to use your license on your new MacBook. You just need to deactivate your license on the old device and reactivate it on the new one. If you forget to deactivate the license on your old MacBook, you can also send a message to our support and we will do it for you.

Best,
Matthias

**Reply by pawanjenu** [reply]
Got it, Thanks

**Reply by thestackdev** [reply]
@MatthiasKerbl 
What if I forgot to deactivate my license on my macbook and format?

**Reply by MatthiasKerbl** [reply]
Hi @thestackdev,

If you forgot to deactivate the license before you reset your MacBook, you can just write our support and we will deactivate the license for you.

Best,
Matthias

**Reply by Nimer-88** [reply]
Is there a way to remove a device from quota?

I mean, I have AlDente installed on my personal computer, my wife computer and a company device. I received new company device and would love to transfer the license to new one. Is that supported?

**Reply by MatthiasKerbl** [reply]
Hi @Nimer-88,

Yes, that's possible; you need to deactivate the license on one of your devices and activate it on the new device.
To deactivate a license on a device, just open the AlDente settings, go to General/Manage License/Deactivate.

Best,
Matthias

**Reply by KolinPower** [reply]
> Hi @thestackdev,
> 
> If you forgot to deactivate the license before you reset your MacBook, you can just write our support and we will deactivate the license for you.
> 
> Best, Matthias

Hello, I can't find the technical support email, please tell me. My question is that I don't remember what license I have (lifetime or PRO) and I want to find out.

**Reply by MatthiasKerbl** [reply]
Hi @KolinPower,

You can text us at news@apphousekitchen.com
Please mention this thread in your email message.

Best,
Matthias

**Comment by seergitx** [comment]
Hi @MatthiasKerbl

About 3 weeks ago, I emailed support to get a license deactivated from my old MacBook (forgot to deactivate before selling). I sent a follow-up email about 2 weeks after that, but still haven't heard anything back.

Definitely checked my spam/junk folders, nada.

It's been a while now, so getting a bit concerned. Any ideas or help would be super appreciated!

**Reply by MatthiasKerbl** [reply]
Hi @seergitx,

Usually, you should have gotten a reply within a few working days. Which email address did you send a request to?

Best,
Matthias

**Reply by seergitx** [reply]
Hi @MatthiasKerbl 

I just got a reply from the team a few days back. They’ve reset the limit for me, so I’m all set! Thanks a bunch for getting back to me!

**Comment by tiiima** [comment]
Hello @MatthiasKerbl

I emailed support for support. I recently switched devices but I had previously used the same license in two other devices, and now it's not letting me use it on my third device. I failed to deactivate the license in the other two devices I had (even though they were both reset and sold). I need help deactivating the license from these devices.

If you could please help me, I'd really appreciate it. Thank you.

**Reply by MatthiasKerbl** [reply]
Hi @tiiima,

Our support will take care of this within a couple of days.
Thank you for being so patient.

Best,
Matthias
